Moments/AddMomentsPage.swift

```diff
@@ -24,33 +24,29 @@ class AddMomentsPage: UIViewController , UITextViewDelegate , UINavigationBarDel
     @IBOutlet weak var datePickerBottomCons: NSLayoutConstraint!
     
```

Comment:
the class name should be *AddMomentViewController* and not "AddMomentsPage", all viewController subclass must have name ends with ViewController